### DISCUSSION TITLE
OpenInv 4.1.8

### DISCUSSION THREAD

**Discussion by Jikoo** [Jikoo/OpenInv#25] [General]
## Supported Minecraft versions
1.16.5, 1.17

## Changelog
* Add 1.17 support

<hr /><em>This discussion was created from the release <a href='https://github.com/Jikoo/OpenInv/releases/tag/4.1.8'>OpenInv 4.1.8</a>.</em>

**Comment by 11mikhailov11** [comment]
please update for 1.18
